bindings/python/setup.py

```diff
@@ -2,15 +2,69 @@
 # encoding: utf-8
 
 from setuptools import setup, Extension
+from pathlib import Path
+import os
+import shutil
+import sys
+import subprocess
+from distutils.command.build import build
+from distutils.command.sdist import sdist
+from setuptools.command.bdist_egg import bdist_egg
 
+ROOT_DIR = Path(os.path.realpath(__file__)).parent
+BUILD_DIR = Path(ROOT_DIR) / "python_build" / "usr"
 
-rust_module = Extension('udbserver',
-                           sources=['udbserver.c'],
-                           libraries=['udbserver'],
-                           )
+VERSION = "0.1"
+
+def cargo_clean():
+    args = ["cargo", "clean"]
+    subprocess.check_output(args, cwd=ROOT_DIR)
+
+def cargo_cinstall():
+    args = ["cargo", "cinstall"]
```

Comment:
```suggestion
    args = ["cargo", "cbuild"]
```
If we not really want to install it, maybe `cbuild` is enough?